resources/qml/TimelineRow.qml

```diff
@@ -133,7 +144,10 @@ Item {
                 Layout.row: 1
                 Layout.column: 0
                 Layout.fillWidth: true
-                Layout.margins: 2
+                Layout.leftMargin: 4
+                Layout.rightMargin: 4
+                Layout.topMargin: reply.visible ? 2 : 4
+                Layout.bottomMargin: Settings.bubbles? 4: 2
                 id: contentItem
```

Comment:
I would like to get rid of the excessive margin-setting. Ideally, the margins would be independent of the bubbles being on or off. Currently, this is a workaround to not increase space use if the bubbles are off. I'd have to give more space to TimeLineRow (probably in MessageView?).

Reply:
I'm not sure I can follow you there, can you try to reword it? Do you want to get rid of all of those 4 lines or not? :D

I'd also probably put the actual message inside a Control: https://doc.qt.io/qt-5/qml-qtquick-controls2-control.html

That way you can set the padding and such depending on if the setting is on and set the margings accordingly. Not sure if that fits into your layout stuff neatly though.

Reply:
I just want to unclutter the margin setting, making it largely independent of whether the bubbles are on or off, mostly to make the code more readable.
A Control seems like it makes sense, my first attempt with it failed, though. I could try again.

Reply:
I would probably just add a padding: 4 on the control to control the sizing of the bubble background. That would a single line dependent on if the bubbles are on or off, which might be fine?

Reply:
I don't want to just add padding in the case of bubbles, because it's not needed everywhere. In some cases, the bubble "border" can just use the space that was already there. And the metadata should use less spacing. So in the end, adding a control might not be the simpler option.

Reply:
No, you're right, it's the cleaner option. I'm adding it now.